rfcs/0185-redistribute-redistributable.md

```diff
@@ -0,0 +1,225 @@
+---
+feature: redistribute-redistributable
+start-date: 2024-12-15
+author: Ekleog
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: https://github.com/NixOS/nixpkgs/issues/83884
+---
+
+# Summary
+[summary]: #summary
+
+Make Hydra build and provide all redistributable software, while making sure installation methods stay as fully free as today.
+
+# Motivation
+[motivation]: #motivation
+
+Currently, Hydra builds only free software and unfree redistributable firmware.
+This means that unfree redistributable software needs to be rebuilt by all the users.
+For example, using MongoDB on a Raspberry Pi 4 (aarch64, which otherwise has access to hydra's cache) takes literally days and huge amounts of swap.
+
+Hydra could provide builds for unfree redistributable software, at minimal added costs.
+This would make life much better for users of such software.
+Especially when the software is still source-available even without being free software, like MongoDB.
+
+# Detailed design
+[design]: #detailed-design
+
+We will add a `runnableOnHydra` field on all licenses, that will be initially set to its `free` field, and set to `true` only for well-known licenses.
```

Comment:
I still feel that doing this per‐package makes more sense given that I think we would want oversight to be on a package‐by‐package basis.